tests/test_assert_assume.cpp

```diff
@@ -0,0 +1,127 @@
+/* Copyright 2019-Barefoot Networks, Inc.
```

Comment:
one more thing: please fix this copyright, unless I'm mistaken and you actually work at Barefoot